packages/AsyncLoader/src/AsyncLoader.unit.mjs

```diff
@@ -153,13 +153,46 @@ test('filters trigger event', async (t) => {
     t.is(errors.length, 0);
 });
 
+test('filters multiple trigger events', async(t) => {
```

Comment:
Titel: 'listens to and filters …' (weil du ja nicht **nur** die Filter testest, sondern auch die mehrfachen Listeners).